packages/website/components/Topbar.tsx

```diff
@@ -93,49 +94,96 @@ export const Topbar = () => (
           <li className={styles.navListItem}>
             <Link href="/" passHref>
               <a className={styles.logoLink} href="/">
-                <Subheading className={styles.navListLink} marginBottom="none">
+                <Text
+                  fontSize="fontSizeL"
+                  lineHeight="lineHeightL"
+                  fontWeight="fontWeightDemiBold"
+                  as="span"
+                  className={styles.navListLink}
+                  marginBottom="none"
+                >
                   Introduction
-                </Subheading>
+                </Text>
               </a>
             </Link>
           </li>
           <li className={styles.navListItem}>
             <Link href="/guidelines/accessibility" passHref>
               <a className={styles.logoLink} href="/guidelines/accessibility">
-                <Subheading className={styles.navListLink} marginBottom="none">
+                <Text
+                  fontSize="fontSizeL"
+                  lineHeight="lineHeightL"
+                  fontWeight="fontWeightDemiBold"
+                  as="span"
+                  className={styles.navListLink}
+                  marginBottom="none"
+                >
                   Guidelines
-                </Subheading>
+                </Text>
               </a>
             </Link>
           </li>
           <li className={styles.navListItem}>
             <Link href="/tokens/color-system" passHref>
               <a className={styles.logoLink} href="/tokens/color-system">
-                <Subheading className={styles.navListLink} marginBottom="none">
+                <Text
+                  fontSize="fontSizeL"
+                  lineHeight="lineHeightL"
+                  fontWeight="fontWeightDemiBold"
+                  as="span"
+                  className={styles.navListLink}
+                  marginBottom="none"
+                >
                   Tokens
-                </Subheading>
+                </Text>
               </a>
             </Link>
           </li>
           <li className={styles.navListItem}>
             <Link href="/components/box" passHref>
               <a className={styles.logoLink} href="/components/box">
-                <Subheading className={styles.navListLink} marginBottom="none">
+                <Text
+                  fontSize="fontSizeL"
+                  lineHeight="lineHeightL"
+                  fontWeight="fontWeightDemiBold"
+                  as="span"
+                  className={styles.navListLink}
+                  marginBottom="none"
+                >
```

Comment:
Last suggest: you could make this a Component so you don't need to repeat this code